GrocifyApp.UI/src/App.jsx

```diff
@@ -55,6 +56,8 @@ function App() {
 
                     <Route path={AppRoutes.ProductSections} element={<PrivateRoute><ProductSections /></PrivateRoute>} />
                     <Route path={AppRoutes.ProductMeaures} element={<PrivateRoute><ProductMeasures /></PrivateRoute>} />
+
+                    <Route path={AppRoutes.ShoppingLists} element={<PrivateRoute><ShoppingLists /></PrivateRoute>} />
```

Comment:
ShoppingLists is the index. Remove weatherForecast and replace it with ShoppingLists

Reply:
or you can keep it, but index should also point to ShoppingLists and not WeatherForecast